package.json

```diff
@@ -5,7 +5,8 @@
   "description": "Foreman isn't really a node module, these are just dependencies needed to build the webpack bundle. 'dependencies' are the asset libraries in use and 'devDependencies' are used for the build process.",
   "private": true,
   "engines": {
-    "node": ">14.0.0 <16.0.0"
+    "node": "^16.0.0",
+    "npm": "^6.0.0"
```

Comment:
I thought Node 16 came with NPM 8 and downgrading this in packaging is really complex, so how hard would it be to not use NPM 6?